t/Net/DRI/Protocol/EPP/OnlyLocalExtensions.t

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+
+use Net::DRI;
+use Net::DRI::Data::Raw;
+
+use Test::More tests => 31;
+
+eval { no warnings; require Test::LongString; Test::LongString->import(max => 100); $Test::LongString::Context=50; };
+if ( $@ ) { no strict 'refs'; *{'main::is_string'}=\&main::is; }
+
+our $E1='<?xml version="1.0" encoding="UTF-8" standalone="no"?><epp xmlns="urn:ietf:params:xml:ns:epp-1.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="urn:ietf:params:xml:ns:epp-1.0 epp-1.0.xsd">';
+our $E2='</epp>';
+our $TRID='<trID><clTRID>ABC-12345</clTRID><svTRID>54322-XYZ</svTRID></trID>';
+
+our ($R1,$R2);
+sub mysend { my ($transport,$count,$msg)=@_; $R1=$msg->as_string(); return 1; }
+sub myrecv { return Net::DRI::Data::Raw->new_from_string($R2? $R2 : $E1.'<response>'.r().$TRID.'</response>'.$E2); }
+sub r { my ($c,$m)=@_; return '<result code="'.($c || 1000).'"><msg>'.($m || 'Command completed successfully').'</msg></result>';  }
+
+my $rc;
+my $dri;
+my $s;
+my $d;
+my ($dh,@c);
+
+####################################################################################################
+
+
+
+###
+# IITCNR (.IT)
```

Comment:
secdns_accredited => 0 should be default, could you add a test here (or in the .IT test) to indicate both logins with and without secdns ?